node/api.go

```diff
@@ -18,6 +19,11 @@ func (node *Node) IsCurrentlyLeader() bool {
 	return node.Consensus.IsLeader()
 }
 
+// GetPublicKeys exposes if node is currently the leader node
+func (node *Node) GetPublicKeys() multibls.PublicKeys {
```

Comment:
Where is this used?

Reply:
This function will be overwritten in next PR https://github.com/harmony-one/harmony/pull/4300/files#diff-b1f5bb00d842600c8142ae84ffba5888cbcc451590063e43198771d4d0b6a1b4R167 . We can keep it now, i'll change it in next pr.